### ISSUE TITLE
Accessibility problem with TAB on register page (Missing order or not set order)

### ISSUE THREAD

**Issue by Tuscann** [podkrepi-bg/frontend#1913]
### Which area(s) of Podkrepi.bg are affected?

Desktop layout, Mobile layout

### Describe the Bug

Accessibility problem with TAB on register page

### To Reproduce

1. Open https://podkrepi.bg/en/register
2. Click with mouse on First Name
3. Click one time tab from keyboard go to Last Name
4. Click one time tab from keyboard go to Email
5. Click one time tab from keyboard go to Password
6. **Click one time tab from keyboard. Nothing happen need to click second time tab from keyboard** to go Confirm password
7. **Click one time tab from keyboard. Nothing happen need to click second time tab from keyboard** to go I agree to the [Terms and Conditions
8. Click one time tab from keyboard go to I understand and I agree to the [General Data Protection Regulation (GDPR)]

Why on place need to click two times Tab but on other place need only one tab? 

### Expected Behavior

Need to go click one tab every time on this page

### Which browser are you using? (if relevant)

Vivaldi

**Comment by dparpulov** [comment]
this is an expected behavior in this case. In the password and confirm password fields there is a button which shows/hides the text when clicked. This is where you are when you click tab while on the password and confirm password fields.

**Comment by Tuscann** [comment]
@dparpulov Did you try step by step to reproduce the bug?

**Comment by dparpulov** [comment]
@Tuscann  Why do you think it's a bug when there's a button which you go to after you click tab once on both fields that has an action when it is clicked -> it shows or hides the text in the field?

**Comment by ElitsaPetrova** [comment]
The real problem is the missing `outline` of the element.

Here is an article [Never remove CSS outlines](https://www.a11yproject.com/posts/never-remove-css-outlines/) that explains why it is not good practice.
